style.css

```diff
@@ -121,3 +122,303 @@ header > * {
+}
+
+.bg-screen-3 {
+  background-image: url(images/device-4.png);
+}
+
+.bg-screen-4 {
+  background-image: url(images/device-3.png);
+}
+
+.bg-screen-5 {
+  background-image: url(images/device-3.png);
+}
+
+.card-details {
+  display: flex;
+  align-items: flex-start;
+  height: 30%;
+  margin-bottom: 0;
+  padding-bottom: 20px;
+  flex-direction: column;
+  flex-wrap: wrap;
+  background: #1a2236;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  backdrop-filter: blur(1px);
+}
+
+.card-details h3 {
+  font-style: normal;
+  font-weight: 300;
+  font-family: 'Poppins', sans-serif;
+  font-size: 22px;
+  margin-left: 20px;
+  line-height: 80%;
+  color: #fff;
+}
+
+.languages {
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: flex-start;
+  font-family: 'Poppins', sans-serif;
+  font-size: 12px;
+  margin-left: 10px;
+  margin-right: 20px;
+}
+
+.language {
+  font-family: 'Poppins', sans-serif;
+  background: #98dede;
+  margin: 10px;
+  padding: 8px;
+  width: 73px;
+  height: 15px;
+  justify-self: center;
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.frw {
+  margin-top: -20px;
+}
+
+.see-this-project {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  align-self: flex-start;
+  width: 200px;
+  height: 50px;
+  border: 1px solid #fff;
+  margin-left: 20px;
+  color: white;
+  margin-top: 20px;
+}
+
+.text {
+  text-align: center;
+  font-weight: bolder;
+  margin-top: 11px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.arrow {
+  font-family: 'Poppins', sans-serif;
+  width: 20.57px;
+  height: 15.35px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 17px;
+}
+
+.about-section-mobile {
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.header .abt {
+  margin-top: 30px;
+  margin-bottom: 30px;
+  font-size: 50px;
+  font-style: normal;
+  font-family: 'Playfair Display', sans-serif;
+  font-weight: 400;
+  line-height: 50px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #1a2236;
+}
+
+.detail {
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 400;
+  font-family: 'Poppins', sans-serif;
+  line-height: 21px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #1a2236;
+}
+
+.experiece {
+  width: 100%;
+  height: 289px;
+  background: #1c528a;
+  font-family: 'Poppins', sans-serif;
+}
+
+ul {
+  list-style: none;
+  width: 50%;
+  margin-left: 40px;
+}
+
+ul li {
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+  text-align: left;
+  color: #fff;
+}
+
+ul li::before {
+  content: "\2022";
+  color: #fff;
+  font-weight: bold;
+  display: inline;
+  width: 1em;
+  margin-right: 20px;
+  margin-left: -1em;
+  text-align: left;
+}
+
+.card-1 {
+  display: flex;
```

Comment:
- [ ] Kindly avoid using the same styling multiple times in your code. Please stick DRY principle in order to make your code clean and crisp. I saw you used `display: flex` style multiple places, kindly use it one time on give it as a class to other wrappers 🙏.